Shared (App)/Shared (Extension)/Resources/src/background/background.ts

```diff
@@ -1,92 +1,76 @@
-`use strict`;
+import { getMessenger, OnMessage } from '../messaging';
 
-(async () => {
-    let method = ``;
+import { getLogger } from '../utils';
 
-    // For message signing:
-    let from = ``;
-    let params = {};
+const log = getLogger('background');
+const Messenger = getMessenger('background', { logger: log });
 
+(async () => {
```

Comment:
No need for this `async` IIFE. Also since the background page isn't persistent we're going to have issues if someone adds an await before `browser.runtime.onMessage.addListener`.

Reply:
You're absolutely right. Will update this weekend.

Reply:
I tried removing the IIFE and both `parcel build` and `parcel watch` worked fine. Do you mind trying again? Parcel already wraps every file in it's own function so to me it's a useless function we can remove and make the code look cleaner.